rest/routing.go

```diff
@@ -373,6 +373,9 @@ func createDiagnosticRouter(sc *ServerContext) *mux.Router {
 	keyspace := r.PathPrefix("/{keyspace:" + dbRegex + "}/").Subrouter()
 	keyspace.StrictSlash(true)
 	keyspace.Handle("/{docid:"+docRegex+"}/_all_channels", makeHandler(sc, adminPrivs, []Permission{PermReadAppData}, nil, (*handler).handleGetDocChannels)).Methods("GET")
+	keyspace.Handle("/sync/{docid:"+docRegex+"}", makeHandler(sc, adminPrivs, []Permission{PermReadAppData}, nil, (*handler).handleSyncFnDryRun)).Methods("GET")
```

Comment:
Our standard approach is use leading underscores for the first entry of keyspaced paths to avoid routing collisions.  I realize there isn't a generic `{docid:"+docRegex+"}` route on the diagnostic router (yet), but I think it would be good to stick to that approach for consistency and future-proofing.